brainscore/benchmarks/imagenet_c.py

```diff
@@ -37,77 +37,77 @@
+        self.stimulus_set = stimulus_set
+        self.noise_types = self.noise_category_map[noise_category]
 
-    def __init__(self, category):
-        category_groups = {
-            'noise': ['gaussian_noise', 'shot_noise', 'impulse_noise'],
-            'blur': ['glass_blur', 'motion_blur', 'zoom_blur', 'defocus_blur'],
-            'weather': ['snow', 'frost', 'fog', 'brightness'],
-            'digital': ['pixelate', 'contrast', 'elastic_transform', 'jpeg_compression']
-        }
-        self._category = category
-        self._groups = category_groups[category]
         ceiling = Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation'])
-        super(Imagenet_C_Category, self).__init__(identifier=f'dietterich.Hendrycks2019-{category}-top1', version=1,
+        super(Imagenet_C_Category, self).__init__(identifier='dietterich.Hendrycks2019-top1', version=1,
                                                   ceiling_func=lambda: ceiling,
                                                   parent='dietterich.Hendrycks2019-top1',
                                                   bibtex=BIBTEX)
 
     def __call__(self, candidate):
         scores = xr.concat([
-            Imagenet_C_Group(group, parent_category=self._category)(candidate)
-            for group in self._groups
+            Imagenet_C_Type(self.stimulus_set, noise_type, self.noise_category)(candidate)
+            for noise_type in self.noise_types
         ], dim='presentation')
-        assert len(set(scores['noise_type'].values)) == len(self._groups)
+        assert len(set(scores['noise_type'].values)) == len(self.noise_types)
         center = np.mean(scores)
         error = np.std(scores)
         score = Score([center, error], coords={'aggregation': ['center', 'error']}, dims=('aggregation',))
         score.attrs[Score.RAW_VALUES_KEY] = scores
         return score
 
-
-class Imagenet_C_Group(BenchmarkBase):
+class Imagenet_C_Type(BenchmarkBase):
     """
     Runs a group in imnet C benchmarks, like gaussian noise [1-5]
     """
-
-    def __init__(self, noise_type, parent_category):
-        self._noise_type = noise_type
+    def __init__(self, stimulus_set, noise_type, noise_category):
+        self.stimulus_set = stimulus_set[stimulus_set['noise_type']==noise_type]
+        self.noise_type = noise_type
         ceiling = Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation'])
-        super(Imagenet_C_Group, self).__init__(identifier=f'dietterich.Hendrycks2019-{noise_type}-top1', version=1,
-                                               ceiling_func=lambda: ceiling,
-                                               parent=f'dietterich.Hendrycks2019-{parent_category}-top1',
-                                               bibtex=BIBTEX)
+        super(Imagenet_C_Type, self).__init__(identifier='dietterich.Hendrycks2019-top1', version=1,
```

Comment:
I believe this should remain at `identifier=f'dietterich.Hendrycks2019-{noise_category}-{noise_type}-top1'` (to be consistent with https://github.com/brain-score/brain-score/pull/267/files#diff-3a908a8c397d945e6f369e56163ca2c22d12a8cbf1d2188a9649a2b3522ec85fR128)